axisregistry/terminal_slope.textproto

```diff
@@ -0,0 +1,19 @@
+#SHARP based on https://github.com/monokromskriftforlag/geologisk/#variable-axis
```

Comment:
Change `SHARP` to `TSLO`.

Reply:
> Is it the final proposition?

No, I haven't updated it with the latest changes. I was waiting for final comments on the issue where it was discussed.
If we agree on the [latest proposal](https://github.com/google/fonts/issues/2849#issuecomment-739977724), and we don't need any further specific approval I would update it, including the suggestion on the description.

Reply:
I've just updated it